package.json

```diff
@@ -133,11 +133,11 @@
     "prettier": "^2.2.1",
     "rimraf": "^3.0.2",
     "sass": "^1.49.9",
-    "sass-loader": "^10.1.1",
+    "sass-loader": "^12.6.0",
```

Comment:
Unfortunately, we cannot use any version of sass-loader above 10.2 because it requires us to use Webpack version 5, and Vue 2 and Nuxt  2 are incompatible with it, as far as I know.
You can see the breaking change regarding the supported Webpack version in `sass-loader`'s Changelog here: https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#-breaking-changes-1